cloudera-integration/csd/scripts/control.sh

```diff
@@ -40,6 +40,24 @@ export LOGBACK_CONFIG=${LOGBACK_CONFIG:-$DEFAULT_LOGBACK_CONFIG}
 export KEYTAB_FILE=${KEYTAB_FILE:-"${CONF_DIR}/pulse.keytab"}
 export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 
+if [ -f COLLECTION_ROLLER_CONFIG ]; then #Checking if file exists
+    if [ ! -s COLLECTION_ROLLER_CONFIG ]; then #Checking if file is not empty
+        export DEFAULT_COLLECTION_ROLLER_CONFIG=$(readlink -f cloudera-integration/csd/aux/default-collection-roller.yml)
```

Comment:
There's no reason to export these values since they are just used in the script. Export is for making variables available to subprocesses